services/token_cancel_airdrop.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
```

Comment:
I just realized the file header information (which helps explain some of the wording) is missing from all of the new files.
The following should be placed _before_ the `syntax` line.
```protobuf
/**
 * # Token Cancel Airdrop
 * Messages used to implement a transaction to cancel a pending airdrop.
 *
 * ### Keywords
 * The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
 * "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this
 * document are to be interpreted as described in [RFC2119](https://www.ietf.org/rfc/rfc2119).
 */
```

Reply:
Added.